submissions/israelOladosu/database/transportation.sql

```diff
@@ -61,24 +50,18 @@ INSERT INTO classes (class_level)
 VALUES(1),(2),(3);
 
 --insert into passangers table
-INSERT INTO passengers (firstname, lastname, sex)
-VALUES('Dele', 'Amid', 'Male'),
-      ('Rita', 'Godspower', 'Female');
+INSERT INTO passengers (name, sex)
+VALUES('Dele Amid', 'Male'),
+      ('Rita Godspower', 'Female');
       
 
 -- insert into trips table
-INSERT INTO trips (passenger_class, passenger_ticket, trip_fare, number_of_parents_children, number_of_siblings_spouse, point_of_embarkation, passenger_id)
-VALUES(1, 'PC174599', 500000, 4, 0, 'Abuja', 1),
-      (1, 'PC56785', 500000, 0, 2, 'Abuja', 2);
-
-
---insert into cabin_classes
-INSERT INTO cabin_classes(cabin_code, trip_id)
-VALUES("C102", 1),
-	  ("C103", 2);
+INSERT INTO trips (passenger_id, passenger_class, passenger_ticket, trip_fare, number_of_parents_children, number_of_siblings_spouse, point_of_embarkation, cabin)
```

Comment:
Kindly remove all the insert query on this file and work with the data given on this current task

Reply:
This is happening because of the update that was made after my task was merged. 
This changes are not meant for the query task, I was just trying to update the second task to suit the changes made to the transportation task.

For the query task I worked with the data given.

Reply:
I know it was for the previous task, Kindly remove them. This insert result together with the data you will be imported for this task will likely affect the result you will be getting